lib/go-tc/users.go

```diff
@@ -260,7 +260,7 @@ type UsersResponseV4 struct {
 	Alerts
 }
 
-// UserResponseV4 is the type of a response from Traffic Ops to requests made
+// UserResponseV4 is the type of response from Traffic Ops to requests made
```

Comment:
you don't need to change this, but just for future reference this was more appropriate with the `a` IMO, since it's describing a strict typing of a particular response rather than the kind or category of a set of related responses.